website/pages/docs/connect/connect-internals.mdx

```diff
@@ -37,13 +37,15 @@ to this, the client provides its own client certificate to show its
 identity to the destination service. If the connection handshake succeeds,
 the connection is encrypted and authorized.
 
-The destination service verifies the client certificate
-against the [public CA bundle](/api/connect/ca#list-ca-root-certificates).
-After verifying the certificate, it must also call the
-[authorization API](/api/agent/connect#authorize) to authorize
-the connection against the configured set of Consul [intentions](/docs/connect/intentions).
-If the authorization API responds successfully, the connection is established.
-Otherwise, the connection is rejected.
+The destination service verifies the client certificate against the [public CA
+bundle](/api/connect/ca#list-ca-root-certificates). After verifying the
+certificate, the next step depends upon the configured application protocol of
+the destination service. TCP (L4) services must authorize incoming connections
+against the configured set of Consul [intentions](/docs/connect/intentions),
+whereas HTTP (L7) services must authorize incoming requests against those same
```

Comment:
```suggestion
the destination service. TCP (L4) services must authorize incoming _connections_
against the configured set of Consul [intentions](/docs/connect/intentions),
whereas HTTP (L7) services must authorize incoming _requests_ against those same
```
Requests vs connections was easy to miss when I read the sentence. I added emphasis so they'd better stand out.